docs/howtos/prometheus.md

```diff
@@ -384,10 +392,22 @@ spec:
 Now, access Grafana by going to `{AMBASSADOR_IP}/grafana/` and logging
 in with `username: admin` : `password: admin`.
 
-Import the [provided dashboard](https://grafana.com/dashboards/10434)
-by clicking the plus sign in the left side-bar, clicking `New
-Dashboard` in the top left, selecting `Import Dashboard`, and entering
-the dashboard ID(10434).
+Before you can import the Ambassador dashboard. You need to add a data source.
+From the Grafana home page, select `Create your first data source`. Now,
+select 'Prometheus'. In the URL section, type in `http://prometheus.default:9090`.
+We deployed prometheus to the default namespace in our example, but if you 
+deployed it to a different namespace, make sure to replace `default` with your
+namespace. Press `Save & Test` to confirm that the data source works.
+
+Import the [provided dashboard](https://grafana.com/grafana/dashboards/4698)
+by clicking the plus sign in the left side-bar, clicking `Import` in the top left, and entering the dashboard ID(4698).
```

Comment:
Why did the dashboard ID change? do we have to worry about it changing again?

Reply:
Not entirely sure. The first time it is mentioned, it links to the updated dashboard which is owned/uploaded by Alex, the second time it is mentioned (and the ID 10434 given) refers to one uploaded by Noah which currently has no overview/description.

Reply:
Yeah, I agree that only one makes sense -- and if we have to take explicit action to make it change, that works out for me. Cool.